app/views/categories/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="nav-head">
+<h2>Transactions for <%= @category.name %></h2>
+<p class="bck"><%= link_to raw('&#8592;'), categories_path %></p>
+</div>
+<div class="container mt-4">
+
+    <p>Total Amount: <%= @total_amount %></p>
+
+    
+    <ul class="list-group">
+      <% @expenses.each do |expense| %>
+        <li class="list-group-item"><%= expense.name %> - Amount: <%= expense.amount %></li>
+      <% end %>
+    </ul>
+  </div>
+
+
+  
+  <div class="new-trans">
+    <%= link_to 'Add a new expense', new_category_expense_path(@category) %>
+</div>
```

Comment:
- [ ] Please consider doing the same with the button for this view 🤓 🔧 

|Template|Your design|
|---|---|
| ![Screenshot from 2023-08-18 21-24-48](https://github.com/eudondian/Budget-App/assets/77069011/bf19c8b5-260a-4207-a0fc-6a09bde260c6) | ![image](https://github.com/eudondian/Budget-App/assets/77069011/f3deb915-8995-4485-b0d5-6e3cff243a81) |